### DISCUSSION TITLE
No documentation available while autocompleting

### DISCUSSION THREAD

**Discussion by krshrimali** [minad/corfu#499] [Q&A]
Hi,

I tried requesting for documentation for autocompletion candidates, but it keeps saying "No documentation available for <>" while there's actually documentation available (at least from LSP):

Image - 1: "No documentation available" for a module named `numpy` in Python
<img width="973" alt="image" src="https://github.com/user-attachments/assets/e55e5664-dc9c-4349-b46b-054f482aa892">

Image - 2: LSP documentation for `numpy`:
<img width="885" alt="image" src="https://github.com/user-attachments/assets/7efd28ea-fdf6-43cd-b582-c62173d5d682">

I've tried this for multiple candidates, even for elisp files, but this doesn't seem to work.

For reference, I'm using doom-emacs, please let me know if this seems to be an issue with doom instead (/if there's a way to debug this forward).

Thanks!

**Comment by minad** [comment]
Hi! in emacs-lisp-mode documentation works for me with and without corfu-popupinfo-mode. Please consider asking on Doomemacs support channels.
